doc/ApiDocGeneration/templates/matthews/styles/main.js

```diff
@@ -7,6 +7,8 @@ WINDOW_CONTENTS = window.location.href.split('/')
 SELECTED_LANGUAGE = 'dotnet'
 BLOB_URI_PREFIX = 'https://azuresdkdocs.blob.core.windows.net/$web?restype=container&comp=list&prefix=dotnet/'
 
+ATTR1 = '[<span class="hljs-meta">System.ComponentModel.EditorBrowsable</span>]\n<'
```

Comment:
What is this for? Is there an issue for it?

Reply:
Yes there is issue [8931](https://github.com/Azure/azure-sdk-for-net/issues/8931)

Reply:
OK in general it is helpful to mention the associated issues when opening the PR so folks can look at the context.